.github/workflows/test.yml

```diff
@@ -36,18 +36,19 @@ jobs:
           - { ruby: '3.0', gemfile: 'rails_7_0' }
           # Ruby 3.1+ has issues with Rails 6.1 https://github.com/rails/rails/issues/46883#issuecomment-1371325906
           # It (Rails 6.1.x), has been marked as a won't fix and as such it's likely this will need to just be excluded until out of support window
-          - { ruby: '3.3', gemfile: 'rails_7_0' }
           - { ruby: '3.3', gemfile: 'rails_7_1' }
```

Comment:
Can we remove this combination too as per our rules it's just 8.all and 7.latest now